packages/frontend/core/src/modules/collection/services/pinned-collection.ts

```
@@ -0,0 +1,71 @@
+
+export class PinnedCollectionService extends Service {
+  constructor(private readonly pinnedCollectionStore: PinnedCollectionStore) {
+    super();
+  }
+
+  pinnedCollections$ = LiveData.from<PinnedCollectionRecord[]>(
+    this.pinnedCollectionStore.watchPinnedCollections(),
+    []
+  );
+
+  sortedPinnedCollections$ = this.pinnedCollections$.map(records =>
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)